README.md

```diff
@@ -46,3 +46,36 @@ Start the container
 
     docker start climatemind-backend
 
+
+## Processing any new Climate Mind OWL ontology file
+
+Follow these instructions to process a new version of a Climate Mind ontology OWL file into files used by Climate Mind (such as Networkx pickle file, test ontology JSON, edge list output CSV, etc). 
+
+1. Download the new OWL ontology file (if haven't already) and make note of the absolute path. DO NOT SAVE the OWL file in the Climate Mind directory in order to avoid accidentally pushing it to the repo.
+
+2. From the climatemind-backend directory run the process_new_ontology_file.py with the absolute path to the new OWL file from step 1. 
+    
+    Example: python3 process_new_ontology_file.py ABSOLUTE_PATH_TO_OWL_FILE
+
+3. New files will be generated and appear in the climatemind-backend directory. These files are also listed in the git.ignore file, so don't worry about them getting pushed accidentally to the git repo.
+
+
+## Visualizing the ontology
+
+Follow these instructions to visualize an OWL file using the Dash dashboard generated by the visualize.py script.
+
+1. Download the new OWL ontology file (if haven't already) and make note of the absolute path. DO NOT SAVE the OWL file in the Climate Mind directory in order to avoid accidentally pushing it to the repo.
+
+2. From the climatemind-backend directory run the process_new_ontology_and_visualize.py with the absolute path to the new OWL file from step 1. 
```

Comment:
Note how to  cd to the directory so others can use who aren't familiar with bash.